src/lib/message/L2ToL1MessageNitro.ts

```diff
@@ -204,16 +204,39 @@ export class L2ToL1MessageReaderNitro extends L2ToL1MessageNitro {
     nodeNum: BigNumber,
     l2Provider: Provider
   ): Promise<ArbBlock> {
-    const node = await rollup.getNode(nodeNum)
+    const { createdAtBlock } = await rollup.getNode(nodeNum)
+
+    let createdFromBlock = createdAtBlock
+    let createdToBlock = createdAtBlock
+
+    const isL2Arbitrum = [42161, 421613, 42170].includes(
+      (await l2Provider.getNetwork()).chainId
+    )
```

Comment:
this can be detected runtime, see https://github.com/OffchainLabs/arbitrum-sdk/pull/319
also we can move this into `getBlockRangesForL1Block `, which is also done in the above PR